src/main/resources/reference.conf

```diff
@@ -7,5 +7,6 @@ qbeast {
     index {
         defaultCubeSize = 100000
         minPartitionCubeSize = 100
+        maxGroupSize = 100000
```

Comment:
This is name cryptic in the config context.

Reply:
Changed it to `cubeWeightsBufferCapacity` in f881e18